server_manager/www/views/server_view/server_metrics_row/bandwidth.ts

```diff
@@ -295,39 +313,53 @@ export class ServerMetricsBandwidthRow extends LitElement {
               ></icon-tooltip>
             </div>
             <div class="bandwidth-container">
-              <span class="bandwidth-percentage"
-                >${this.formatPercentage(this.bandwidthPercentage)}</span
-              >
-              <span class="bandwidth-fraction"
-                >${formatBytes(this.totalBytes, this.language)}
-                /${formatBytes(this.limitBytes, this.language)}</span
-              >
-              <span class="bandwidth-progress-container">
-                <progress
-                  max=${this.limitBytes}
-                  value=${this.totalBytes}
-                ></progress>
-                <icon-tooltip
-                  text="${this.hasDataLimits
-                    ? ''
-                    : this.localize(
-                        'server-view-server-metrics-bandwidth-limit-tooltip'
-                      )}"
-                  icon="warning"
-                ></icon-tooltip>
-              </span>
+              ${this.metrics.dataTransferred
+                ? html`<span class="bandwidth-percentage"
+                      >${this.formatPercentage(this.bandwidthPercentage)}</span
+                    >
```

Comment:
Is a tool auto-formatting these like this with the line breaks inside the closing tags etc.?

```suggestion
                ? html`
                <span class="bandwidth-percentage">
                    ${this.formatPercentage(this.bandwidthPercentage)}
                </span>
```